demo/docs/features/metas.md

```diff
@@ -0,0 +1,43 @@
+# Metas
+
+Metas can be seen as labels. They are additional data that accompany the errors, events etc. in order to filter them
+better.
+
+## Description
+
+### Client
+
+Grafana JavaScript Agent requires `app` meta as the only mandatory meta when initializing the library. Besides `app`,
```

Comment:
The mandatory or auto-created metas could be listed as a list or table with a brief description of them. Would make it a bit more scannable.

Reply:
Sounds good!